src/components/analyticVisualization/EditVisualization.js

```diff
@@ -0,0 +1,96 @@
+import React, {useEffect, useState} from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import {
+    Button,
+    ButtonStrip,
+    Modal,
+    ModalActions,
+    ModalContent,
+    ModalTitle,
+} from '@dhis2/ui'
+import {NotEditableElement, VisualizationGroup, VisualizationTitle} from "./index";
+
+const EditVisualization = ({open, settings, handleChange, currentGroup, groups, handleEdit, handleClose, type}) => {
+    const [typeName, useTypeName] = useState('')
+    const [label, useLabel] = useState('')
+
+    const chooseTypeName = type => {
+        switch (type) {
+            case 'home':
+                useTypeName('')
+                useLabel('Home')
```

Comment:
I think these strings need to be translated as they're passed through as the label of the NonEditableItem below?

Reply:
Thanks Tom. I'm adding this:

```
'home': i18n.t('Home'),
'program': i18n.t('Program'),
'dataset': i18n.t('Data set')
```